lighthouse-core/test/audits/seo/tap-targets-test.js

```diff
@@ -0,0 +1,184 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* eslint-env jest */
+
+const TapTargetsAudit = require('../../../audits/seo/tap-targets.js');
+const assert = require('assert');
+
+function auditTapTargets(tapTargets) {
+  const artifacts = {
+    TapTargets: tapTargets,
+    MetaElements: [{
+      name: 'viewport',
+      content: 'width=device-width',
```

Comment:
There are a few cases that still need covering, e.g. need viewport failure test(s). See the red lines in https://coveralls.io/builds/21238984/source?filename=lighthouse-core%2Faudits%2Fseo%2Ftap-targets.js#L42